*** Nessuna regola per generare l'obiettivo «matcher/reftime/reftime-lex.cc», necessario per «distdir».  Stop.

sui branch master e cpp11
autoreconf -if
./configure
make dist
make  dist-gzip am__post_remove_distdir='@:'
make[1]: Entering directory `/root/git/arkimet'
if test -d "arkimet-0.81"; then find "arkimet-0.81" -type d ! -perm -200 -exec chmod u+w {} ';' && rm -rf "arkimet-0.81" || { sleep 5 && rm -rf "arkimet-0.81"; }; else :; fi
test -d "arkimet-0.81" || mkdir "arkimet-0.81"
 (cd conf && make  top_distdir=../arkimet-0.81 distdir=../arkimet-0.81/conf \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[2]: Entering directory `/root/git/arkimet/conf'
make[2]: Leaving directory `/root/git/arkimet/conf'
 (cd embedded && make  top_distdir=../arkimet-0.81 distdir=../arkimet-0.81/embedded \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[2]: Entering directory `/root/git/arkimet/embedded'
 (cd wibble/wibble && make  top_distdir=../../../arkimet-0.81 distdir=../../../arkimet-0.81/embedded/wibble/wibble \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[3]: Entering directory `/root/git/arkimet/embedded/wibble/wibble'
make[3]: Leaving directory `/root/git/arkimet/embedded/wibble/wibble'
 (cd sqlite && make  top_distdir=../../arkimet-0.81 distdir=../../arkimet-0.81/embedded/sqlite \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[3]: Entering directory `/root/git/arkimet/embedded/sqlite'
make[3]: Leaving directory `/root/git/arkimet/embedded/sqlite'
 (cd minilzo && make  top_distdir=../../arkimet-0.81 distdir=../../arkimet-0.81/embedded/minilzo \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[3]: Entering directory `/root/git/arkimet/embedded/minilzo'
make[3]: Leaving directory `/root/git/arkimet/embedded/minilzo'
make[2]: Leaving directory `/root/git/arkimet/embedded'
 (cd arki && make  top_distdir=../arkimet-0.81 distdir=../arkimet-0.81/arki \
     am__remove_distdir=: am__skip_length_check=: am__skip_mode_fix=: distdir)
make[2]: Entering directory `/root/git/arkimet/arki'
make[2]: *** Nessuna regola per generare l'obiettivo «matcher/reftime/reftime-lex.cc», necessario per «distdir».  Stop.
make[2]: Leaving directory `/root/git/arkimet/arki'
make[1]: *** [distdir] Errore 1
make[1]: Leaving directory `/root/git/arkimet'
make: *** [dist] Errore 2

--- a/arki/Makefile.am
+++ b/arki/Makefile.am
@@ -171,17 +171,16 @@ nobase_dist_arkiinclude_HEADERS = \
 		utils/net/http.h \
 		values.h
 
-BUILT_SOURCES = $(top_builddir)/arki/matcher/reftime/reftime-lex.h \
-		$(top_builddir)/arki/matcher/reftime/reftime-lex.cc \
-		$(top_builddir)/arki/matcher/reftime/reftime-parse.hh \
-		$(top_builddir)/arki/matcher/reftime/reftime-parse.cc
+BUILT_SOURCES = \
+		$(srcdir)/matcher/reftime/reftime-lex.h \
+		$(srcdir)/matcher/reftime/reftime-lex.cc \
+		$(srcdir)/matcher/reftime/reftime-parse.hh \
+		$(srcdir)/matcher/reftime/reftime-parse.cc
 
-all_built_sources: $(BUILT_SOURCES)
-
-$(top_builddir)/arki/matcher/reftime/reftime-lex.h $(top_builddir)/arki/matcher/reftime/reftime-lex.cc: $(srcdir)/matcher/reftime/reftime-lex.ll
+$(srcdir)/matcher/reftime/reftime-lex.h $(srcdir)/matcher/reftime/reftime-lex.cc: $(srcdir)/matcher/reftime/reftime-lex.ll
 	$(srcdir)/buildflex $<
 
-$(top_builddir)/arki/matcher/reftime/reftime-parse.hh $(top_builddir)/arki/matcher/reftime/reftime-parse.cc: $(srcdir)/matcher/reftime/reftime-parse.yy
+$(srcdir)/matcher/reftime/reftime-parse.hh $(srcdir)/matcher/reftime/reftime-parse.cc: $(srcdir)/matcher/reftime/reftime-parse.yy
 	$(srcdir)/buildbison $<
 
 lib_LTLIBRARIES = libarkimet.la
@@ -618,5 +617,7 @@ tests_arki_test_LDADD += libarkimet.la $(SQLITE3_LDFLAGS) $(DEVEL_LIBS) -lpthrea
 EXTRA_DIST = runtest buildbison buildflex \
 	     matcher/reftime/reftime-lex.ll \
 	     matcher/reftime/reftime-parse.yy \
-	     $(top_builddir)/arki/matcher/reftime/reftime-lex.h \
-	     $(top_builddir)/arki/matcher/reftime/reftime-parse.hh
+	     $(srcdir)/matcher/reftime/reftime-lex.h \
+	     $(srcdir)/matcher/reftime/reftime-lex.cc \
+	     $(srcdir)/matcher/reftime/reftime-parse.hh \
+	     $(srcdir)/matcher/reftime/reftime-parse.cc

